DolbyIO/Source/Private/DolbyIOObserver.cpp

```diff
@@ -48,13 +48,15 @@ void UDolbyIOObserver::FwdOnDisconnected()
 {
 	BroadcastEvent(OnDisconnected);
 }
-void UDolbyIOObserver::FwdOnParticipantAdded(const FDolbyIOParticipantInfo& ParticipantInfo)
+void UDolbyIOObserver::FwdOnParticipantAdded(const EDolbyIOParticipantStatus Status,
```

Comment:
I removed const on by-value arguments because I don't think it makes much sense. Perhaps in definitions, but almost certainly not in declarations. But feel free to leave it as is and resolve this if you want. Otherwise also change the update event.

Reply:
I remember your comment on UE conventions so I followed it ;)